packages/grid/x-data-grid/src/components/columnHeaders/ColumnHeaderMenuIcon.tsx

```diff
@@ -47,22 +47,28 @@ export const ColumnHeaderMenuIcon = React.memo((props: ColumnHeaderMenuIconProps
 
   return (
     <div className={classes.root}>
-      <rootProps.components.BaseIconButton
-        ref={iconButtonRef}
-        tabIndex={-1}
-        className={classes.button}
-        aria-label={apiRef.current.getLocaleText('columnMenuLabel')}
+      <rootProps.components.BaseTooltip
         title={apiRef.current.getLocaleText('columnMenuLabel')}
-        size="small"
-        onClick={handleMenuIconClick}
-        aria-expanded={open ? 'true' : undefined}
-        aria-haspopup="true"
-        aria-controls={columnMenuId}
-        id={columnMenuButtonId}
-        {...rootProps.componentsProps?.baseIconButton}
+        enterDelay={1000}
```

Comment:
Is this delay a little bit above the user's expectation? As it's supposed to work in place of the default HTML tooltip which I guess [is](https://ux.stackexchange.com/a/119975) 0.3-0.5 seconds, do you think the delay should be set to a smaller number?

The MUI [Tooltip](https://mui.com/material-ui/react-tooltip/) also has a default delay of 0.1s and [material design](https://m2.material.io/components/tooltips#behavior shows a minor delay in the image.

The same (1s) is the case for the filter button tooltip though (maybe there was a reason behind that which I am not aware of 🤔 )

Reply:
Yep, I used the same delay that was being used for the filter button tooltip.
I'm not sure why this exact value was used, maybe to simulate the behavior of `aria-label` that is not shown immediately, but after a delay?

Reply:
Probably somewhere around the default delay for HTML `title` attribute (0.3-0.5 seconds) would be closer to a normal user's expectation.
Not a big deal though. Being consistent with the filter button tooltip is also fine.

Reply:
> Probably somewhere around the default delay for HTML title attribute (0.3-0.5 seconds) would be closer to a normal user's expectation.

Maybe it depends on the browser, but in Chromium, it seems to be more than 0.5s:

https://user-images.githubusercontent.com/13808724/230153497-676d38e4-efe2-49f3-9174-19bbe693381f.mov